### ISSUE TITLE
Kingdom Hearts 1.5 ReMix (BLUS31212) sound issues

### ISSUE THREAD

**Issue by JacobAL** [RPCS3/rpcs3#3217]
Sorry if this is a known issue. I couldn't see it in the open category.
Sometimes while playing 1.5 ReMix, the sound will just stop completely. It doesn't restart on entering a new area. Need to restart emulation in order to get sound back.
It happens entirely randomly. Seems to happen more often if I've been playing longer, but that might just be because it's more likely to happen the longer I play.

[Specs](http://i.imgur.com/TXkVWPh.png)
I'd post the log for the error, but the last log I made was too big to easily open. If it happens again the next time I play, I'll try to post it.

**Comment by JacobAL** [comment]
[There's an example of it happening at 2h47m40s in this video.](https://mixer.com/Speckxz?vod=6413777) 
It may be during loading screens/area transitions that it's more likely to happen.
[I doubt this helps much, but here's a screenshot of the log directly after closing and restarting the game.](http://i.imgur.com/a0eNCxn.png)

**Comment by scribam** [comment]
@JacobAL Does it still happen with a recent build?

**Comment by JacobAL** [comment]
Haven't tested in recent builds.

**Comment by Sirius902** [comment]
This issue also happens to me in Kingdom Hearts 2.5 ReMix on the most recent build as of now being 6514. 

Windows 10 Creators Update
Intel(R) Core(TM) i7-6700 CPU
NVIDIA GeForce GTX 980 Ti

[Sound Cutting Out RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/1809189/Sound.Cutting.Out.RPCS3.log.gz)

**Comment by hectoryo** [comment]
So I have been experiencing this annoying bug too, and I kept updating my emulator daily. Today, I think, after #4422 my sound issue seems to be fixed. I've played around 1 hour and a half without the audio crashing, and after that I've tried to launch the game 5 times (before this pull, when I loaded my savegame it had a high chance to crash the audio) and the sound was nice.
Anyway keep in mind that some regressions seems to be happening with that pull (example #4473 ) and fixing those regressions could end in our audio not working again.
